summit_blood_samples/blood_sample/templates/upload-processed.html

```diff
@@ -5,6 +5,10 @@ <h3>1-Upload Processed</h3>
             <div class="form-group">
                 <label for="processedfile">Please select csv file: <abbr style="color:#c71d1d" title="Required">*</abbr></label>
                 <input type="file" required class="form__control form-control-file" id="processedfile" accept=".csv">
+                <label for="siteheld">Site held:</label>
+                <select name="siteheld" id="siteheld" class="form__control ">
+                    <option value="0">UK Biocentre</option>
```

Comment:
It's not ideal that the options are hardcoded here rather than being pulled from the model choices.